dist/diffhtml-runtime.js

```diff
@@ -341,6 +368,11 @@ function getFinalizeCallback(node, state) {
 
     // Dispatch an event on the node once rendering has completed.
     node.dispatchEvent(new CustomEvent('renderComplete'));
+
+    // Call the remaining middleware with a completion Date.
+    remainingMiddleware.forEach(function (f) {
+      return f(new Date());
```

Comment:
Yeah, really wish GitHub respected my `.gitattributes` to suppress diffing these files.